sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -131,6 +131,7 @@ dependencies {
   compile library.java.arrow_memory_core
   compile library.java.arrow_vector
 
+  compile "io.opencensus:opencensus-api:0.28.0"
```

Comment:
This version introduces the log4shell vulnerability. It should be updated to the latest